src/model/OutputControlFiles.cpp

```diff
@@ -0,0 +1,628 @@
+}
+
+bool OutputControlFiles::setOutputDelightIn(bool outputDelightIn) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDelightIn(outputDelightIn);
+}
+
+bool OutputControlFiles::setOutputDelightELdmp(bool outputDelightELdmp) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDelightELdmp(outputDelightELdmp);
+}
+
+bool OutputControlFiles::setOutputDelightDFdmp(bool outputDelightDFdmp) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDelightDFdmp(outputDelightDFdmp);
+}
+
+bool OutputControlFiles::setOutputEDD(bool outputEDD) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputEDD(outputEDD);
+}
+
+bool OutputControlFiles::setOutputDBG(bool outputDBG) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDBG(outputDBG);
+}
+
+bool OutputControlFiles::setOutputPerfLog(bool outputPerfLog) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputPerfLog(outputPerfLog);
+}
+
+bool OutputControlFiles::setOutputSLN(bool outputSLN) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputSLN(outputSLN);
+}
+
+bool OutputControlFiles::setOutputSCI(bool outputSCI) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputSCI(outputSCI);
+}
+
+bool OutputControlFiles::setOutputWRL(bool outputWRL) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputWRL(outputWRL);
+}
+
+bool OutputControlFiles::setOutputScreen(bool outputScreen) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputScreen(outputScreen);
+}
+
+bool OutputControlFiles::setOutputExtShd(bool outputExtShd) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputExtShd(outputExtShd);
+}
+
+bool OutputControlFiles::setOutputTarcog(bool outputTarcog) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputTarcog(outputTarcog);
+}
+
+/// @cond
+OutputControlFiles::OutputControlFiles(std::shared_ptr<detail::OutputControlFiles_Impl> impl)
+  : ModelObject(std::move(impl))
+{}
+OutputControlFiles::OutputControlFiles(Model& model)
+  : ModelObject(OutputControlFiles::iddObjectType(),model)
+{
+  // This is a required-field, so pick one...
+  // Same defaults as EnergyPlus IDD
+  setOutputCSV(false);
+  setOutputMTR(false);
+  setOutputESO(true);
+  setOutputEIO(true);
+  setOutputTabular(true);
+  setOutputSQLite(true);
+  setOutputJSON(true);
+  setOutputAUDIT(true);
+  setOutputZoneSizing(true);
+  setOutputSystemSizing(true);
+  setOutputDXF(true);
+  setOutputBND(true);
+  setOutputRDD(true);
+  setOutputMDD(true);
+  setOutputMTD(true);
+  setOutputEND(true);
+  setOutputSHD(true);
+  setOutputDFS(true);
+  setOutputGLHE(true);
+  setOutputDelightIn(true);
+  setOutputDelightELdmp(true);
+  setOutputDelightDFdmp(true);
+  setOutputEDD(true);
+  setOutputDBG(true);
+  setOutputPerfLog(true);
+  setOutputSLN(true);
+  setOutputSCI(true);
+  setOutputWRL(true);
+  setOutputScreen(true);
+  setOutputExtShd(true);
+  setOutputTarcog(true);
```

Comment:
Ok, this matches the defaults in the IDD: all true, except the Output MTR (for some reason).

Reply:
MTR and CSV